src/api/parser/test/e2e/parser-flow.test.js

```diff
@@ -8,28 +8,7 @@ const getWikiFeeds = require('../../src/utils/wiki-feed-parser');
 
 const urlToId = (url) => hash(normalizeUrl(url));
 
-jest.mock('../../src/utils/wiki-feed-parser');
-
-const valid = [
-  {
-    author: 'Tue Nguyen',
-    url: 'https://dev.to/feed/tuenguyen2911_67',
-  },
-  {
-    author: 'Antonio Bennett',
-    url: 'https://dev.to/feed/antoniobennett',
-  },
-];
-const invalid = [
-  {
-    author: 'Waqas Khan',
-    url: 'http://wkhan10.wordpress.com/feed',
-  },
-  {
-    author: 'Daniel Hodgin',
-    url: 'http://www.hodgin.ca/?feed=rss2&cat=4',
-  },
-];
+jest.mock('../../src/utils/wiki-feed-parser', () => jest.fn());
```

Comment:
Tells The mock factory returns a mocked function so it wouldn't look for `__mocks__`